project.clj

```diff
@@ -25,13 +25,13 @@
                  [javax.servlet/javax.servlet-api "3.1.0"]
 
                  ;; Jetty Webserver
-                 [org.eclipse.jetty/jetty-server "9.2.8.v20150217"
+                 [org.eclipse.jetty/jetty-server "9.2.10.v20150310"
                   :exclusions [org.eclipse.jetty.orbit/javax.servlet]]
-                 [org.eclipse.jetty/jetty-servlet "9.2.8.v20150217"]
-                 [org.eclipse.jetty/jetty-servlets "9.2.8.v20150217"]
-                 [org.eclipse.jetty/jetty-webapp "9.2.8.v20150217"]
-                 [org.eclipse.jetty/jetty-proxy "9.2.8.v20150217"]
-                 [org.eclipse.jetty/jetty-jmx "9.2.8.v20150217"]
+                 [org.eclipse.jetty/jetty-servlet "9.2.10.v20150310"]
+                 [org.eclipse.jetty/jetty-servlets "9.2.10.v20150310"]
+                 [org.eclipse.jetty/jetty-webapp "9.2.10.v20150310"]
+                 [org.eclipse.jetty/jetty-proxy "9.2.10.v20150310"]
+                 [org.eclipse.jetty/jetty-jmx "9.2.10.v20150310"]
```

Comment:
That thought has crossed my mind a few times.  Not sure if we did it this way just so we were more intentional about understanding what deps we were going to be pulling in.  Each time I've updated Jetty, I've downloaded the Jetty source and run mvn deps on it to confirm that all of these match up for a given release.  Maybe I've been too paranoid there.

I'm open to this change, though.  @cprice404 - what do you think?